server/data/data-cassandra/src/main/java/org/apache/james/rrt/cassandra/CassandraRRTModule.java

```diff
@@ -37,5 +39,14 @@
             .addPartitionKey(CassandraRecipientRewriteTableTable.USER, text())
             .addClusteringColumn(CassandraRecipientRewriteTableTable.DOMAIN, text())
             .addClusteringColumn(CassandraRecipientRewriteTableTable.MAPPING, text()))
+        .table(CassandraMappingsSourcesTable.TABLE_NAME)
+        .comment("Holds mapping sources.")
```

Comment:
We have to e explaining it is a projection table for retrieving sources associated mappings.